frontend/src/components/pages/UserDashboard/index.jsx

```diff
@@ -42,10 +48,11 @@ function DashboardUnconnected({
 
 const mapStateToProps = (state) => {
   return {
-    users: state.apiEntities.users || {},
-    userDetailedStats: state.apiEntities.userDetailedStats || {},
+    users: state.apiEntities.users,
+    userDetailedStats: state.apiEntities.userDetailedStats,
+    userBurndownStats: state.apiEntities.burndownSnapshots,
```

Comment:
Why is userBurnDownStats here?  Wasn't it removed on purpose?

If you search for userBurnDownStats in this file then you will see that it is not being used.